### ISSUE TITLE
Question about Xiaomi Mi 360º Home Security Camera Pro 2K and lens blocking

### ISSUE THREAD

**Issue by ScorpSoul** [al-one/hass-xiaomi-miot#1285]
Hi,
sorry if this question is not allowed here, but I'd like to know if it's possible to trigger lens blocking on this addon on my Xiaomi Mi 360º Home Security Camera Pro 2K.

I'd like to set it up according to other sensors on HomeAssistant to block/unblock the lens when I'm around the office.

Thanks

**Comment by simongrech** [comment]
Yes should be possible. Install prev version as the latest version is broken. There is this bug reported to get it fixed. https://github.com/al-one/hass-xiaomi-miot/issues/1330
